manifests/init.pp

```diff
@@ -133,6 +133,33 @@
 #   If set to false, a restart will be executed instead of a reload.
 #   This default will be changed in a future release.
 #
+#########
+######### REWRITE
+#
+# [*external_repo*]
```

Comment:
I think we should use puppet-strings style documentation for new parameters and not javadoc (or however this is called)

Reply:
did I miss a style change?